deepsea.spec

```diff
@@ -273,66 +272,90 @@ sed -i '/^master_minion:/s!_REPLACE_ME_!'`hostname -f`'!' /srv/pillar/ceph/maste
 %postun 
 
 %files
-%defattr(-,salt,salt,-)
-/srv/modules/pillar/*
-%dir /etc/salt
-%dir /etc/salt/master.d
+%defattr(-,root,root,-)
+/srv/modules/pillar/stack.py
+%dir %attr(0750, root, salt) %_sysconfdir/salt
```

Comment:
I believe they are, and hope we can resolve this better with a future bug report and patch.